meteor/client/ui/TestTools/RecordingsList.tsx

```diff
@@ -0,0 +1,290 @@
+			deleteConfirmItem: item,
+			showDeleteConfirm: true
+		})
+	}
+	handleConfirmDeleteAccept = (e) => {
+		if (this.state.deleteConfirmItem) {
+			Meteor.call(ClientAPI.methods.execMethod, eventContextForLog(e), TestToolsAPI.methods.recordDelete, this.state.deleteConfirmItem._id)
+		}
+		this.setState({
+			showDeleteConfirm: false
+		})
+	}
+
+	isStudioConfigured () {
+		const { studio } = this.props
+		if (!studio) return false
+
+		const config = objectPath.get(studio, 'testToolsConfig.recordings')
+		if (!config) return false
+
+		if (!config.channelIndex || !config.decklinkDevice || !config.deviceId) return false
+
+		return true
+	}
+
+	renderControlPanel () {
+		const { t } = this.props
+
+		if (!this.isStudioConfigured()) {
+			return <React.Fragment>
+				<p>{t('A required setting is not configured')}</p>
+			</React.Fragment>
+		}
+
+		const active = this.props.files.find(f => !f.stoppedAt)
+
+		let obj = this.state
+		return <React.Fragment>
+			<p>Status: {active ? t('Active') : t('Ready')}</p>
+			<p>Name: {active ? active.name : <EditAttribute
+				obj={obj}
+				updateFunction={this.onUpdateValue}
+				attribute='filename'
+				type='text'
+			/>}</p>
+			<p>Started: {active ? <MomentFromNow>{active.startedAt}</MomentFromNow> : '-'}</p>
+			<p>
+				{
+					active
+						? <button onClick={e => this.stopRecording(e)}>{t('Stop')}</button>
+						: <button onClick={e => this.startRecording(e)}>{t('Start')}</button>
+				}
+			</p>
+		</React.Fragment>
+	}
+	renderRecordingList () {
+		return this.props.files.map((file) => (
+			<RecordedFilesListItem key={file._id} file={file} onDeleteRecording={i => this.onDelete(i)} />
+		))
+	}
+
+	render () {
+		const { t } = this.props
+
+		// console.log('obj', obj)
+		return <React.Fragment>
+			<div className='mtl gutter'>
+				<header className='mvs'>
+					<h1>{t('Recordings')}</h1>
+				</header>
+				<div className='mod mvl'>
+					{this.renderControlPanel()}
+				</div>
+				<ModalDialog title={t('Delete this item?')} acceptText={t('Delete')} secondaryText={t('Cancel')} show={this.state.showDeleteConfirm} onAccept={(e) => this.handleConfirmDeleteAccept(e)} onSecondary={(e) => this.handleConfirmDeleteCancel(e)}>
+					<p>{t('Are you sure you want to delete recording "{{name}}"?', { name: this.state.deleteConfirmItem && this.state.deleteConfirmItem.name })}</p>
+					<p>{t('Please note: This action is irreversible!')}</p>
+				</ModalDialog>
+				<div className='mod mvl'>
+					<table className='table system-status-table expando expando-tight'>
+						<thead>
+							<tr className='hl'>
+								<th className='c3'>
+									{t('Name')}
+								</th>
+								<th className='c2'>
+									{t('Started')}
+								</th>
+								<th className='c2'>
+									{t('Stopped')}
+								</th>
+								<th className='c1'>
+								</th>
+							</tr>
+						</thead>
+						<tbody>
+							{this.renderRecordingList()}
+						</tbody>
+					</table>
+				</div>
+			</div>
+		</React.Fragment>
```

Comment:
Like before, just return the `div`. I don't think we expect to return any elements at the same level as the `div`.